charts/common/README_CHANGELOG.md.gotmpl

```diff
@@ -10,6 +10,12 @@ All notable changes to this application Helm chart will be documented in this fi
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [3.1.0]
+
+#### Added
+
+- Allow setting secret values through Helm templates.
```

Comment:
This could be misconstrued as that the value for `secrets` implements the Helm `tpl` function which isn't the case. Maybe something like this?

```suggestion
- Allow setting secret values through Helm values file.
```